samples/main.cpp

```diff
@@ -69,8 +69,11 @@ int main(int /*argc*/, char* /*argv*/[])
   auto console = NovelRT::LoggingService(NovelRT::Utilities::Misc::CONSOLE_LOG_APP);
   auto audio = runner.getAudioService();
   audio.lock()->initializeAudio();
-  //auto bgm = audio->load((soundsDirPath / "running.ogg").string(), true);
-  //auto jojo = audio->load((soundsDirPath / "jojo.ogg").string(), false);
+  auto bgm = audio.lock()->loadMusic((soundsDirPath / "marisa.ogg").string());
```

Comment:
What is the behavior if the file cant be found?

Reply:
Previously it would throw an error - I've since corrected this to specify the file that cannot be found and to also let the user know _why_ the file couldn't be loaded (in a case of i/o error, etc).

Also, added support so that if the service detects the buffer wasn't created, it'll warn and return instead of killing the engine.